src/main/kotlin/org/jitsi/jibri/selenium/pageobjects/CallPage.kt

```diff
@@ -171,7 +171,7 @@ class CallPage(driver: RemoteWebDriver) : AbstractPageObject(driver) {
     fun leave(): Boolean {
         val result = driver.executeScript("""
             try {
-                return APP.conference._room.connection.disconnect();
+                return APP.conference._room.leave();
```

Comment:
this was i tweak i made to better exit the meeting and make sure we send the presence-unavailable when we were going to handle that on the web to make it look like jibri left sooner.  i don't think we're going to end up doing that, but this change is probably still something we should have.